concepts/string.md

```diff
@@ -0,0 +1,53 @@
+---
+description: String
+---
+
+# String
+
+A string is a sequence of characters.
+In computer programming, strings are used to represent text.
+
+By default, a string is _plain text_.
+It does not represent rich formatting or styles, like font-size and color.
+
+- A string with no characters is the _empty string_.
+- A string can contain even just a single character: `a`
+- A string can contain any character even outside the English alphabet, including emojis: `A man with a telescope 🔭 from the city of Třebíč`
+- A string can be very long (like paragraphs, books) and can contain control characters like _newline_:
+   ```
+   Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+   
+   Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+   ```
+
+## Encoding
+
+A particular focus is paid to how the characters are encoded in the string.
+The computer does not know what letters are, only numbers.
+Therefore every character needs to have some number assigned (codepoint).
+
+### ASCII
+
+The ASCII standard is a widely popular codetable that maps English and other particularly useful characters to numbers.
```

Comment:
As a general rule, let's avoid words like "popular", because they can go out of date, although in this case it probably will be true for a long time.

Here "widely" and "particularly" can be removed.

Try not to stuff too much information into one sentence, and avoid ambiguous parses.

> The ASCII standard maps English letters and other common characters to numbers.